docusaurus/docs/reference/tunnelers/linux/index.md

```diff
@@ -187,34 +184,55 @@ repo_gpgcheck=1
 
 2. Run `sudo yum update` to refresh your repo data cache. Optionally, you may wish to also install all available updates.
 2. Run `sudo yum install ziti-edge-tunnel` to install the RPM.
-2. Place a copy of the enrollment token JWT file in the `/opt/openziti/etc/identities` directory.
-2. Set the filemode and owner so that members of group `ziti` will be able to read and delete the token file.
-
-  ```text
-  sudo chown -cR :ziti        /opt/openziti/etc/identities
-  sudo chmod -cR ug=rwX,o-rwx /opt/openziti/etc/identities
-  ```
-
 2. Enable and start the service
 
     ```text
     sudo systemctl enable --now ziti-edge-tunnel.service
     ```
 
-2. The process needs to be restarted if the contents of `/opt/openziti/etc/identities` change.
+2. Add an Identity
 
-    ```text
-    sudo systemctl restart ziti-edge-tunnel.service
-    ```
+## Adding an Identity
+
+### Restart to Load All in Identity Directory
+
+The tunneler will load all identities in the `--identity-dir` directory. The identities directory created by the package
+installation is `/opt/openziti/etc/identities`. You may add identity files to this directory by copying the identity
+file into the directory and setting permissions for group `ziti`.
+
+```text
+sudo chown -cR :ziti        /opt/openziti/etc/identities
+sudo chmod -cR ug=rwX,o-rwx /opt/openziti/etc/identities
+```
+
+The tunneler process needs to be restarted if the contents of `/opt/openziti/etc/identities` change.
+
+```text
+sudo systemctl restart ziti-edge-tunnel.service
+```
+
+### Load a Single Identity Without Restarting
+
+This sends an IPC command to the running tunneler. You must have write permission on the IPC socket
```

Comment:
"you" wording shold be avoided. as should 'options'  we should just state:

"to add an identity `ziti-edge-tunnel` should be run as the user it's executing as. By default this user is `ziti`. Add another identity by running:

Reply:
this doc has __numerous__ you/your types of references that really should be cleaned up imo